pkgs/development/tools/misc/blisp/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  IOKit,
+  testers,
+  blisp,
+}:
+let
+  version = "0.0.3";
+in stdenv.mkDerivation {
+  pname = "blisp";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "pine64";
+    repo = "blisp";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-m2ETPugSqOVrYNWLYwY5xYuReBImUyfm4jTyAoaqS0k=";
+  };
+
+  nativeBuildInputs = [cmake];
```

```suggestion
  nativeBuildInputs = [ cmake ];
```
we like spaces in nixpkgs :D